change/@ni-nimble-components-d9a02072-800c-4be6-943b-b8d354302001.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
What's the plan for updating clients? Will you do it? File bugs? Promote at UI WG?